pkg/api/mlflow/dao/repositories/metric.go

```diff
@@ -110,8 +110,8 @@ func (r MetricRepository) CreateBatch(
 	}
 
 	for n := range metrics {
-		metrics[n].Context = *allContexts[n]
 		metrics[n].ContextID = allContexts[n].ID
+		metrics[n].Context = *allContexts[n]
```

Comment:
Why are there no tests here?

Reply:
the `repositories` functions are generally tested by the integration tests

Reply:
Well, this is changing functionality, but there is no corresponding test.

Reply:
there's an extensive test for this area in `tests/integration/golang/mlflow/run/log_batch_test.go` -- i'll try to add something for this change